src/overlays/actors/ovl_En_Stream/z_en_stream.c

```diff
@@ -14,10 +14,12 @@ void EnStream_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnStream_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnStream_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnStream_Draw(Actor* thisx, GlobalContext* globalCtx);
+void EnStream_WaitForPlayer(EnStream* this, GlobalContext* globalCtx);
+void EnStream_SuckPlayer(EnStream* this, GlobalContext* globalCtx);
+s32 func_80B0B81C(PosRot* vortexPosRot, PosRot* playerPosRot, Vec3f* v, f32 vortexYScale);
 
 extern UNK_TYPE D_06000950;
```

Comment:
Based on the usage of this variable in the draw function, we can update the type to a display list (ie. a `Gfx[]`)
```suggestion
extern Gfx D_06000950[];
```